example/test_problem/Hydro/CCSN/Input__Parameter.Leakage

```diff
@@ -0,0 +1,307 @@
+MG_MAX_ITER                  -1           # maximum number of iterations in multigrid: (<0=auto) [-1]
+MG_NPRE_SMOOTH               -1           # number of pre-smoothing steps in multigrid: (<0=auto) [-1]
+MG_NPOST_SMOOTH              -1           # number of post-smoothing steps in multigrid: (<0=auto) [-1]
+MG_TOLERATED_ERROR           -1.0         # maximum tolerated error in multigrid (<0=auto) [-1.0]
+POT_GPU_NPGROUP              -1           # number of patch groups sent into the CPU/GPU Poisson solver (<=0=auto) [-1]
+OPT__GRA_P5_GRADIENT          0           # 5-points gradient in the Gravity solver (must have GRA/USG_GHOST_SIZE_G>=2) [0]
+OPT__GRAVITY_EXTRA_MASS       0           # add extra mass source when computing gravity [0]
+OPT__SELF_GRAVITY             1           # add self-gravity [1]
+OPT__EXT_ACC                  0           # add external acceleration (0=off, 1=function, 2=table) [0] ##HYDRO ONLY##
+OPT__EXT_POT                  3           # add external potential    (0=off, 1=function, 2=table, 3=GREP) [0]
+                                          # --> for 3 (GREP), one must also enable GREP in the Makefile
+                                          # --> for table/GREP, edit the corresponding parameters below too
+
+
+# source terms
+SRC_DELEPTONIZATION           1           # deleptonization (for simulations of stellar core collapse) [0] ##HYDRO ONLY##
+SRC_DELEP_ENU                10.0         # deleptonization parameter E_nu in MeV [10.0] ##HYDRO ONLY##
+SRC_DELEP_RHO1                4.9316595742e7  # deleptonization parameter rho_1 in g/cm^3 [3.0e7 ] ##HYDRO ONLY##
+SRC_DELEP_RHO2                1.5882185396e13 # deleptonization parameter rho_2 in g/cm^3 [2.0e13] ##HYDRO ONLY##
+SRC_DELEP_YE1                 0.5             # deleptonization parameter Ye_1 [0.5  ] ##HYDRO ONLY##
+SRC_DELEP_YE2                 0.265508866     # deleptonization parameter Ye_2 [0.278] ##HYDRO ONLY##
+SRC_DELEP_YEC                 0.0601082693    # deleptonization parameter Ye_c [0.035] ##HYDRO ONLY##
+SRC_LIGHTBULB                 0           # lightbulb scheme for neutrino heating and cooling [0] ##HYDRO ONLY##
+SRC_LIGHTBULB_LNUE            4.0e52      # electron neutrino luminosity in erg/s [1e52] ##HYDRO ONLY##
+SRC_LIGHTBULB_TNUE            4.0         # electron neutrino temperature in MeV [4.0] ##HYDRO ONLY##
+SRC_LEAKAGE                   0           # multiflavour neutrino leakage scheme [0]
+SRC_LEAKAGE_NRADIUS           768         # number of bin in the radial    direction [768]
+SRC_LEAKAGE_NTHETA            1           # number of bin in the polar     direction [1]
+SRC_LEAKAGE_NPHI              1           # number of bin in the azimuthal direction [1]
+SRC_LEAKAGE_BINSIZE_RADIUS    1.0         # bin size of linear bins in radius [1.0]
+SRC_LEAKAGE_RADIUSMAX         3.0e3       # maximum radius [3.0e3]
+SRC_LEAKAGE_RADIUSMIN_LOG     3.0e2       # minimum radius of logarithmic bins [3.0e2]
```

Comment:
units are km?

Reply:
These values are in code unit, which is set to 1 km (`UNIT_L`). I will update it to cm later.

Reply:
Updated. Now the three input parameters (`BinSize`, `RadiusMax`, and `RadiusMin_Log`) are in cm.